feat(blog): add "Keeping Perspective" blog post (#48)
spokestack/spokestack-website: content/blog/keeping-perspective.md

---
title: Keeping Perspective
date: '2020-03-19T13:17:39.257Z'

> description: Spokestack is a powerful platform of open source libraries and robust services to make

Reviewer: Description looks cut off.

author: mike
draft: false
---

Since 2012, I watch [this presentation](https://vimeo.com/34017777) by Wilson Minor at the 2011 Build Conference at least once a year. I watched it again last week as a way of trying to refocus myself on our product development work in the midst of all the stress with our current pandemic. I find watching Wilson’s talk helps to not only reset my thinking about why we make things but the consequences of making things - even if they are digital and potentially ephemeral. If you can spare 38 minutes, it's worth your time.

> Nine years ago, before we knew the full impact of how mobile and social services might affect our lives, Wilson referenced a quote that not only addressed these (at the time) nascent products but everyday products we already take for granted. This quote in his presentation from Marshall McLuhan was this:

Reviewer 2: ```suggestion
Nine years ago, before we knew the full impact of how mobile and social services might affect our lives, Wilson referenced a quote that not only addressed these (at the time) nascent products but everyday products we already take for granted. The quote was from Marshall McLuhan:
```

> > We Shape Our Tools, and Thereafter Our Tools Shape Us

Reviewer 2: ```suggestion
> We shape our tools, and thereafter our tools shape us.
```

> Whether a direct quote from McLuhan or an adaption from a [Churchill quote on rebuilding England’s House of Commons building](https://quoteinvestigator.com/2016/06/26/shape/), this adage is a reminder that the things we build have an impact on the world. Even if it [looks like a toy at first](https://cdixon.org/2010/01/03/the-next-big-thing-will-start-out-looking-like-a-toy), like [Chris Dixon](https://twitter.com/cdixon) said 10 years ago. Sometimes at the beginning of a project, you may not fully understand all the potential uses for what you are building. But if you’re lucky and keep your eyes and ears open, some of those uses may find you.

Reviewer 2: ```suggestion
Whether a direct quote from McLuhan or an adaptation from a [Churchill quote on rebuilding England’s House of Commons building](https://quoteinvestigator.com/2016/06/26/shape/), this adage is a reminder that the things we build have an impact on the world. Even if it [looks like a toy at first](https://cdixon.org/2010/01/03/the-next-big-thing-will-start-out-looking-like-a-toy), like [Chris Dixon](https://twitter.com/cdixon) said 10 years ago. Sometimes at the beginning of a project, you may not fully understand all the potential uses for what you are building. But if you’re lucky and keep your eyes and ears open, some of those uses may find you.
```

At [Spokestack](https://www.spokestack.io), we build tools and services that allow companies to talk to their customers on their mobile devices. The idea started with “wouldn’t it be cool if you could talk to your apps?”. That came from the frustration as a user of asking Siri questions and getting the “this is what we found on the web” answers back. Sometimes, the answers were already on our phone in the apps we download, locked behind taps and gestures when they could easily have a voice user interface.

> As we have spoken with different groups about how our service might help companies, one of the most common features is better accessibility for users who need more than a touchscreen to use their app. There are already a few companies that build hardware specifically for visually impaired users. There are also companies like [Tecla](https://gettecla.com/pages/tecla-e) that build devices to help those with limited upper-body mobility to use mobile devices. The more we’ve started to research this area, the more we realized that we could enable better usability for the millions of people who could benefit from a voice interface for mobile apps.

Reviewer 2: ```suggestion
As we have spoken with different groups about how our service might help companies, one of the most commonly mentioned features is better accessibility for users who need more than a touchscreen to use their app. There are already a few companies that build hardware specifically for visually impaired users. There are also companies like [Tecla](https://gettecla.com/pages/tecla-e) that build devices to help those with limited upper-body mobility to use mobile devices. The more we’ve started to research this area, the more we've realized that we could enable better usability for the millions of people who could benefit from a voice interface for mobile apps.
```

> New companies are always looking for product-market fit for all the obvious business reasons. However, in times like these it can fill overwhelming trying to build a business and not know how it might help the overwhelming needs of society.

Reviewer 2: ```suggestion
New companies are always looking for product-market fit for all the obvious business reasons. However, in times like these it can feel overwhelming trying to build a business and not knowing how it might help the overwhelming needs of society.
```

We have a ways to go before we can know how our work might impact people’s lives. So as we shape our tools, we are going to keep thinking about how we might address better accessibility to users in a number of different stories.

> If you have any ideas or specific problems where our voice interface experience can improve you or your users’ lives, please reach out to us and help us shape Spokestack into a better tool.

Reviewer 2: ```suggestion
If you have any ideas or specific problems where our voice interface experience can improve your or your users’ lives, please reach out to us and help us shape Spokestack into a better tool.
```
Should "reach out to us" be a link?
